root/layout/components/Footer.js

```diff
@@ -1,95 +1,520 @@
-          `Brought to you by {MeB|MetaBrainz Foundation} and our
-           {spon|sponsors} and {supp|supporters}. Cover Art provided
-           by the {caa|Cover Art Archive}.`,
-          {
-            caa: '//coverartarchive.org/',
-            MeB: 'https://metabrainz.org/',
-            spon: 'https://metabrainz.org/sponsors',
-            supp: 'https://metabrainz.org/supporters',
-          },
-        )}
-
-        {DBDefs.DB_STAGING_SERVER && DBDefs.GIT_BRANCH ? (
-          <>
+    <section className="bs footer">
+      <div className="ms-4 me-4 mb-3 ps-4 pb-3 pe-4 border-top container">
+        <div className="row mb-4 mt-2 pt-2">
+          <div className="col-sm-12 col-md-4">
+            <h3 className="ms-3">
+              <img
+                alt="MusicBrainz"
+                src={logo}
+                width="196"
+              />
+            </h3>
             <br />
-            {exp.l('Running: {git_details}', {
-              git_details: (
-                <span
-                  className="tooltip"
-                  key="git_details"
-                  title={DBDefs.GIT_MSG}
-                >
-                  {DBDefs.GIT_BRANCH}
-                  {' '}
-                  {bracketedText(DBDefs.GIT_SHA)}
+            <ul className="list-unstyled ms-3">
+              <li>
+                <span className="fs-5">
+                  {l('Development IRC:')}
+                </span>
+                <a
+                  className="fw-bold fs-5 ms-1"
+                  href="https://kiwiirc.com/nextclient/irc.libera.chat/?#metabrainz"
+                  rel="noopener noreferrer"
+                  target="_blank"
+                >
+                  {l('#metabrainz')}
+                </a>
+              </li>
+              <li>
+                <span className="fs-5">
+                  {l('Discussion IRC:')}
                 </span>
-              ),
-            })}
-          </>
-        ) : null}
+                <a
+                  className="fw-bold fs-5 ms-1"
+                  href="https://kiwiirc.com/nextclient/irc.libera.chat/?#musicbrainz"
+                  rel="noopener noreferrer"
+                  target="_blank"
+                >
+                  {l('#musicbrainz')}
+                </a>
+              </li>
+              <li>
+                <span className="fs-5">
+                  {l('Email:')}
+                </span>
+                <a
+                  className="fw-bold fs-5 ms-1"
+                  href="mailto:support@metabrainz.org"
+                >
+                  {l('support@metabrainz.org')}
+                </a>
+              </li>
+              <li>
+                <a
+                  href="https://twitter.com/MusicBrainz"
+                  rel="noreferrer"
+                  target="_blank"
+                >
+                  <FontAwesomeIcon
+                    icon={faTwitter}
+                    size="lg"
+                  />
+                </a>
```

Comment:
These look good, but they are not accessible, unless I'm missing something (someone using a screen reader will have no idea what is going on). See https://savvas.me/accessibility/problem-with-social-icons for a suggestion on how to improve it while still keeping the icons :) 

Also, should they be a tiny bit bigger? They feel small compared with the surrounding stuff.

More generally, while linking to Twitter makes sense, does it make sense to link to the others? Our latest Instagram post is from 2018. If we offer links here, that kinda suggests we're going to be actually *using* these and responding to people there (we kinda-sorta do answer on Facebook, although we don't use it much at all).

Reply:
Fixed! I have moved them to the right as well.

Reply:
I don't have a strong opinion on where to put the social links, but I wouldn't stuff them all into a single bullet point (they should be their own `<ul>` I guess).